pkg/webtests/caldav_test.go

```diff
@@ -318,3 +321,76 @@ END:VCALENDAR`
 		assert.Contains(t, rec.Body.String(), "RELATED-TO;RELTYPE=PARENT:uid-caldav-test-parent-task-another-list")
 	})
 }
+
+func TestCaldavReport(t *testing.T) {
```

Comment:
Nit: Doesn't this test the _projects_ report response? Then the test method should be called accordingly.